config/config.js

```diff
@@ -81,19 +81,23 @@ exports.config = {
     disabled  : true
   },
 
-  /*
-   * PS3 support is supplied by Gimx and is available on Windows and Linux
-   * only.  Refer to the README for information on setup.
-   */
   ps3 : {
+    deviceNotes: [
```

Comment:
OH, I really really like your idea of extending the config object and adding an elaboration class / layer. So each field / object would be responsible of communicating how it would (if it does) want to be rendered / seen by the user! I thought about what happens upon config save and yeah, perhaps a page refresh is in order, and then each field also communicating if a server restart is required? Something like updateConfig.js could then invoke a restart procedure via Node-to-bash commands and listen for std err / out to see if the restart has completed? Good stuff. :)